components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1743,6 +1743,8 @@ interface ZCashWalletService {
       (ZCashAccountInfo? account_info);
   ValidateZCashAddress(string addr, bool testnet) =>
       (ZCashAddressValidationResult result);
+  ShieldFunds(string network_id, AccountId account_id) =>
+      (string? tx_id, string? error_message);
```

Comment:
will this always return an error since `enable_orchard` is disabled for iOS for now?

Reply:
Yes, this will just return an error. Same behavior when feature flag is disabled

Reply:
in this case, maybe we dont need to expose this api for iOS just yet until the [linkage](https://github.com/brave/brave-browser/issues/39582) is resolved. and Android is using WebUI.